### DISCUSSION TITLE
Integrate spark-lineage with data in S3

### DISCUSSION THREAD

**Discussion by mykola-yesypchuk-inflection** [open-metadata/OpenMetadata#16095] [General]
Hello. 
I am looking for the data lineage solution and trying to integrate OpenMetadata with my data pipeline. 
Currently I checking how [spark-lineage](https://docs.open-metadata.org/v1.3.x/connectors/ingestion/lineage/spark-lineage) and [s3 storage](https://docs.open-metadata.org/v1.3.x/connectors/storage/s3) are working together, but it seems not working. After my pipeline run I can see that UI get "spark pipeline entry" but empty lineage threads inside it.
Any suggestion?

**Comment by harshach** [comment]
@ulixius9 can you chime in here

**Comment by ulixius9** [comment]
hey @mykola-yesypchuk-inflection, just a couple of days back we added the support for s3 container lineage with spark agent as part of this pr: https://github.com/open-metadata/openmetadata-spark-agent/pull/17

we will soon be releasing this fix, once that's out you would be able to see the lineage between tables and containers 🙏

**Reply by mykola-yesypchuk-inflection** [reply]
Great news. Thanks!
